pkg/apis/v1alpha1/vpcnetworkconfiguration_types.go

```diff
@@ -25,10 +25,13 @@ type VPCNetworkConfigurationSpec struct {
 	EdgeClusterPath string `json:"edgeClusterPath,omitempty"`
 	// NSX-T Project the Namespace associated with.
 	NSXTProject string `json:"nsxtProject,omitempty"`
-	// NSX-T IPv4 Block paths used to allocate public Subnets.
+	// Flag to indicate whether support for load balancing is needed
+	// +kubebuilder:default=false
+	LoadBalancerVPCEndpointEnabled bool `json:"loadBalancerVPCEndpointEnabled,omitempty"`
```

Comment:
VPC Endpoint is a term in AWS with a different meaning, so let us change a name unless NSX-T VPC uses it.

How about CreateLoadBalancerSubnet if it is just about creating a Subnet for AVI SE?

Reply:
And we should keep the T1 networking behavior to create the Subnet using the reserved CIDR for SEs, not allocating from VPC Private CIDR. Please update the commit message for that.

Reply:
in NSX VPC API, term "LoadBalancerVPCEndpoint" is used in VPC payload, we keep consistent with NSX VPC API.

Reply:
Confirmed with Avi team that the AVI SE subnet will be created from VPC Private IPBlock. If LoadBalancerVPCEndpoint enabled set to true, when creating VPC, a Private subnet will be auto-created for Avi SE.